spec/plans/execute.fmf

```diff
@@ -318,3 +291,54 @@ description: |
                     systemctl start httpd
                     echo foo > /var/www/html/index.html
                     curl http://localhost/ | grep foo
+
+/sync:
+    summary: Synchronization between steps or tests
+    description: |
+        The main idea of :ref:`/spec/plans/provision/multihost`
+        tests is to run certain tasks in parallel (e.g., a RestAPI
+        requests from a client to a server). Therefore, these kinds
+        of tests may require test synchronization.
+        The synchronization may be specified for specific steps
+        by providing a list of step names under ``step`` keyword.
+        This creates a synchronization
+        barrier before each provided step.
+
+        There is also an option for a synchronization on the test
+        level defined using ``test`` keyword accepting several values:
+
+            each
+                Synchronize execution of all tests.
+            names
+                A list of test names executed on both hosts that
+                should be synchronized before running.
+            indexes
+                A list of indexes into the list of tests, which labels
+                specific tests for synchronization.
+            none
+                **Default** value, which disables the synchronization
+                on the test level.
+    example:
+      - |
+        execute:
+            how: tmt
+            sync:
+                step:
+                  - prepare
+                  - execute
+                test: each
+
+      - |
+        execute:
+            how: tmt
+            sync:
+                step:
+                  - prepare
+                test:
+                    names:
+                      - /basic/test1
+                      - /basic/test2
+                      - /core/bfs
+                    indexes:
```

Comment:
```suggestion
                    indices:
```

Reply:
Fixed in https://github.com/teemtee/tmt/pull/1706. However, the both forms should be correct [0].

[0] https://www.nasdaq.com/articles/indexes-or-indices-whats-the-deal-2016-05-12

Reply:
I'm afraid there's still something to fix, you changed "indexes" to "ind**e**ces", not "ind**i**ces"...

Reply:
The currently proposed keyword is `index`. 

See https://github.com/teemtee/tmt/issues/1705#issuecomment-1326348409